pkg/config/config.go

```diff
@@ -821,6 +821,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.pid_cache_size", 10000)
 	config.BindEnvAndSetDefault("runtime_security_config.cookie_cache_size", 100)
 	config.BindEnvAndSetDefault("runtime_security_config.agent_monitoring_events", true)
+	config.BindEnvAndSetDefault("runtime_security_config.runtime_enabled", false)
```

Comment:
How do we ensure that, after the upgrade of the agent, all the rules won't be ignored ?

Reply:
I'm going to change the whole logic, it was just to introduce the parameters, but per internal discussion we need to handle them differently